environment.yml

```diff
@@ -13,9 +13,9 @@ dependencies:
   - tqdm
   - scikit-image=0.*
   - scikit-gstat>=1.0
-  - geoutils=0.1.*
+  - geoutils>=0.1.2
   - pip
 
-  # To run CI against latest GeoUtils
-  # - pip:
-  #   - git+https://github.com/GlacioHack/geoutils.git
+   # To run CI against latest GeoUtils
+#  - pip:
+#    - git+https://github.com/rhugonnet/geoutils.git@fix_to_points
```

Comment:
Should we not remove these lines?

Reply:
It's good to keep them commented for CI testing of the same kind (not having to release a version in GeoUtils to check it passes CI in xDEM). They served me well here! :smile: